### PR TITLE
Update react template generator config

### PR DESCRIPTION
**Pull request by tar-z**

### CODE REVIEW

File: templates-generator/react-config.js [hunk 1]
```diff
@@ -18,16 +18,16 @@ module.exports = {
             glob: 'src/app-routes.js',
             definitions: [
                 {
-                    before: 'import',
-                    after: '<%=^empty%>import'
+                    before: /import {([\s\w,]*)} from '.\/pages'/,
```
Comment (egaluza) [inline templates-generator/react-config.js:21]: To make regular expressions of the same type use "import {([^}]*)} from '.\/pages'"

File: templates-generator/react-config.js [hunk 2]
```diff
@@ -18,16 +18,16 @@ module.exports = {
...
+                    after: '<%=^empty%>import {$1} from \'./pages\''
                 },
                 {
-                    before: '];',
-                    after: '<%=/empty%>];'
+                    before: 'const routes = [',
```
Comment (egaluza) [inline templates-generator/react-config.js:25]: Use: "before:  '(const routes = [)'"

File: templates-generator/react-config.js [hunk 3]
```diff
@@ -18,16 +18,16 @@ module.exports = {
...
+                    after: '<%=/empty%>const routes = [<%=^empty%>'
```
Comment (egaluza) [inline templates-generator/react-config.js:26]: Use: "after: '<%=/empty%>$1[<%=^empty%>'"

File: templates-generator/react-config.js [hunk 4]
```diff
@@ -18,16 +18,16 @@ module.exports = {
             glob: 'src/app-routes.js',
             definitions: [
                 {
-                    before: 'import',
-                    after: '<%=^empty%>import'
+                    before: /import {([^}]*)} from '.\/pages'/,
+                    after: '<%=^empty%>import {$1} from \'./pages\''
```
Comment (nightskylark) [inline templates-generator/react-config.js:22]:
```suggestion
                    before: /(import {[^{]*'.\/pages'\N\n)/,
                    after: '<%=^empty%>$1<%=/empty%>'
```

File: templates-generator/react-config.js [hunk 5]
```diff
@@ -18,16 +18,16 @@ module.exports = {
...
                 },
                 {
-                    before: '];',
-                    after: '<%=/empty%>];'
+                    before: /(const routes = \[)/,
+                    after: '<%=/empty%>$1<%=^empty%>'
```
Comment (nightskylark) [inline templates-generator/react-config.js:26]:
```suggestion
                    before: /const routes = \[([^]]*)\]/,
                    after: '<%=^empty%>$1<%=/empty%>'
```

File: templates-generator/react-config.js [hunk 6]
```diff
@@ -18,16 +18,16 @@ module.exports = {
...
                 },
                 {
-                    before: 'export default [',
-                    after: '<%=/empty%>export default [<%=^empty%>'
+                    before: '];',
+                    after: '<%=/empty%>];'
```
Comment (nightskylark) [inline templates-generator/react-config.js:30]:
```suggestion
```

Comment (nightskylark) [inline templates-generator/react-config.js:30]: And this block can be removed

**Pull request merged**